public/styles.css

```diff
@@ -181,13 +181,92 @@
 }
 
 .qaList {
-  height: 200px;
+  height: 250px;
   overflow: scroll;
-  border-style: solid;
 }
 
+<<<<<<< HEAD
```

Comment:
I'm not sure if this is part of the current markup on github or if this is still in the file... If we could double check before I merge tomorrow that would be sweet. Referring to the '<<<<< HEAD' writing on line 188.